server/GlideAction.ts

```diff
@@ -0,0 +1,9 @@
+declare class GlideAction {
+  /**
+   * Set getRedirectURL in UI Actions.
+   * @param scope takes the 'current' object.
+   */
+  setRedirectURL<TRecord>(scope: TRecord): void;
```

Comment:
```suggestion
  setRedirectURL(scope: GlideRecord<any> | string): void;
```

Reply:
This method can take a string as well. Shouldn't need to make it generic since the GlideRecord type should cover the GlideRecord use case

Reply:
Good spot, thanks. I updated the description and the name of the method parameter to 'target' as well. Sound more clear to me.